src/image/Image.cc

```diff
@@ -119,7 +110,7 @@ ImageBase<PixelT>::ImageBase(ImageBase const& rhs, bool const deep
 }
 // Delegate to copy-constructor for backwards compatibility
 template <typename PixelT>
-ImageBase<PixelT>::ImageBase(ImageBase&& rhs) : ImageBase(rhs, false) {}
+ImageBase<PixelT>::ImageBase(ImageBase&& rhs) noexcept : ImageBase(rhs, false) {}
```

Comment:
If this is delegating to the copy-constructor, it probably can't guarantee it's not throwing.